Source/SlashCmd/Main.lua

```diff
@@ -0,0 +1,50 @@
+local _, addonTable = ...;
+local zc = addonTable.zc;
+
+function Auctionator.SlashCmd.Initialize()
+  SlashCmdList["Auctionator"] = Auctionator.SlashCmd.Handler
+  SLASH_Auctionator1 = "/auctionator"
+  SLASH_Auctionator2 = "/atr"
+end
+
+function Auctionator.SlashCmd.Handler(input)
+  Auctionator.Debug.Message( 'Auctionator.SlashCmd.Handler', msg )
+
+  local command = {zc.words(input:lower())};
+
+  if #command == 0 then
+    return
+  end
+  if command[1] == "reset" or command[1]=="r" then
+    Auctionator.SlashCmd.Reset(command[2])
+  elseif (command[1] == "toggle" or command[1]=="t") then
+    Auctionator.SlashCmd.Toggle(command[2])
+  else
+    Auctionator.Utilities.Message("Unrecognized command")
+  end
+end
+
+function Auctionator.SlashCmd.Reset(identifier)
```

Comment:
If we are planning on adding more commands, we might want a dispatch table instead?
```lua
local AUCTIONATOR_COMMANDS = {
  "t" = Auctionator.Variables.ResetTimer,
  "timer" = Auctionator.Variables.ResetTimer,
  "db" = Auctionator.Variables.ResetDatabase,
  "database" = Auctionator.Variables.ResetDatabase
}

function Auctionator.SlaschCmd.Reset(identifier, ...)
  local handler = AUCTIONATOR_COMMANDS[identifier]

  if handler == nil then
    Auctionator.Utilities.Message("Unrecognized command '" .. identifier .. "'")
  else
    handler(...)
  end
end
```
We would then put the messaging in with the command handler. Makes it a little more straightforward to add commands.